qt/widgets/common/inc/MantidQtWidgets/Common/FitPropertyBrowser.h

```diff
@@ -443,6 +443,11 @@ private slots:
   /// Clear the Fit status display
   void clearFitResultStatus();
 
+public: // functions declared public for testing
+  QString getOldExpressionAsString(QString &parameterName) const;
```

Comment:
I think the parameter can be const as well here and in the .cpp file ie `const QString &parameterName`